setup.cfg

```diff
@@ -21,3 +21,62 @@ include_package_data = True
 [options.entry_points]
 console_scripts =
     relenv = relenv.__main__:main
+
+[sdist]
+owner = root
```

Comment:
Is this definitely needed? https://github.com/pypa/setuptools/pull/2800

Reply:
yes, I believe `root` is not the default user. It'll use which ever user is creating the tarball.